README.md

```diff
@@ -19,6 +19,11 @@ If you have an idea how to improve our MVVM Framework, please contact us via our
 
```

Comment:
Can you update the following lines (above)?

Line 7:
`* The DevExpress.Mvvm.Free version is designed for use with the standard or third-party controls.`

Please remove the article ("the") before standard:

`* The DevExpress.Mvvm.Free version is designed for use with standard or third-party controls.`

Line 18:
`If you have an idea how to improve our MVVM Framework, please contact us via our [support center](https://www.devexpress.com/Support/Center/Question/Create).`

Can you update this sentence as follows?

`If you have an idea on how to improve our MVVM Framework, please contact us at our [Support Center](https://www.devexpress.com/Support/Center/Question/Create).`